reconcile/test/runtime/fixtures.py

```diff
@@ -0,0 +1,49 @@
+from typing import (
```

Comment:
Maybe a personal taste, but I would use a `conftest.py` instead of this file and use the [pytest fixtures](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html) feature to consume `TestIntegration` or `ShardableTestIntegration` instances in my test cases

Reply:
fixed in e6e3d6453535b1f29bc7615f1ee065c45fa06c64